internal/backend/backend.go

```diff
@@ -250,6 +250,8 @@ func (b *Backend) getUserID(ctx context.Context, username string, password []byt
 		})
 
 		return "", ErrLoginBlocked
+	} else {
+		logrus.Errorf("Incorrect login credentials. Please try again, Attempt %d", count)
```

Comment:
That matches the old log the CS relies on.
+ after 3 failed attempt you get blocked so this could lead to better understanding once the user get blocked